portfolio/src/main/java/com/google/sps/servlets/AuthenticateServlet.java

```diff
@@ -32,31 +33,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     int status = 0;
     Gson gson = new Gson();
-    ArrayList<String> toReturn = new ArrayList<String>();
-
-    String redirectTo = "/index.html";
+    String redirectTo = "/index.html#comments";
     response.setContentType("application/json");
-
     UserService userService = UserServiceFactory.getUserService();
+    
+    ArrayList<String> toReturn = new ArrayList<String>();
```

Comment:
I would recommend defining an authentication response class, and returning an object of that class instead of using an array like this. The array approach requires the caller to know how this API is implemented, i.e. it must know what each value at each index corresponds to. Returning a well structured json object does not, and makes the response easy to extend in the future.